velox/expression/tests/ExpressionFuzzerTest.cpp

```diff
@@ -61,7 +61,9 @@ int main(int argc, char** argv) {
       "in",
       "element_at",
       "width_bucket",
-  };
+      // This function argument is a serialized bloom filter, it cannot accept
+      // random data.
+      "might_contain"};
```

Comment:
might_contain is a Spark function, but this test only uses Presto functions. Please, remove.